.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+/Dockerfile @exercism/ops
```

Comment:
I think the `.dockerignore` file should also be added. And what about the `CODEOWNERS` file itself?

Reply:
Why should .dockerignore be added?

Reply:
I now know this from your PR to the Ruby one. Thanks :)

Reply:
:)

Reply:
Should I merge or hold out on this?